src/web/mjs/engine/Settings.mjs

```diff
@@ -57,9 +57,10 @@ export default class Settings extends EventTarget {
             input: types.select,
             options: [
                 { value: 'frontend@classic-light', name: 'Classic (Light)' },
-                { value: 'frontend@classic-dark', name: 'Ken\'s Daedal Dark' }
+                { value: 'frontend@classic-dark', name: 'Ken\'s Daedal Dark' },
+                { value: 'frontend@modern', name: 'Modern <devemloop>' }
             ],
-            value: 'frontend@classic-light'
+            value: 'frontend@modern'
```

Comment:
I think we should stay on classic-light by default -for a while- so that more "early adopters" can test it thouroughly.